modules/apps/remote-app/remote-app-support-web/test/index.js

```diff
@@ -196,7 +196,7 @@ describe('remote-app-support-web', () => {
 
 			await reply();
 
-			expect(receiveMessage).toHaveBeenCalledTimes(1);
+			expect(receiveMessage).toHaveBeenCalled();
```

Comment:
So how many times is it called now? It would be good to understand exactly what's coming in on the next call(s) and see whether it's indicative of a problem.